src/pages/hyunjinKim/Main/MainKim.js

```diff
@@ -1,8 +1,64 @@
 import React, { Component } from 'react';
-//import { withRouter } from 'react-router';
+import Aside from './Aside/Aside';
+import Nav from './Nav/Nav';
+import '@fortawesome/fontawesome-free/js/all.js';
+import Feeds from './Feeds/Feeds';
+import './MainKim.scss';
+import '../../../styles/common.scss';
+import '../../../styles/reset.scss';
+
 class MainKim extends Component {
+  constructor() {
+    super();
+    this.state = {
+      feeds: [],
+      recommands: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/feedData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          feeds: data,
+        });
+      });
+
+    fetch('http://localhost:3000/data/recommandData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          ...this.state,
+          recommands: data,
+        });
+      });
```

Comment:
```suggestion
      .then(data => {
        this.setState({ recommands: data });
      });
```
이렇게 하는거랑 같은 동작일거 같아요

Reply:
앗 네 수정했습니다!